src/components/JSONEdit.tsx

```diff
@@ -39,26 +42,39 @@ const JSONEdit: React.FC<JSONEditProps & TextareaHTMLAttributes<HTMLTextAreaElem
     let updatedData: any;
     try {
       updatedData = JSON.parse(rawValue);
-      setJSONValid(true);
-    } catch {
-      setJSONValid(false);
+      setJsonLogicError('');
+    } catch (error) {
+      setJsonLogicError(error.toString());
```

Comment:
is this properly localized?

Reply:
No. But neither are the JSON.parse() errors. Discussed it with Joeri. L10n could be something for the future.